.env

```diff
@@ -1,3 +1,2 @@
-Dont remove this is need for the eslint (eslint-plugin-import) plugin to work
-
-The error will be something like this https://gist.github.com/kaareal/7a9eec01e6f86aa855678541201ae638
+# Dont remove this is need for the eslint (eslint-plugin-import) plugin to work
+# The error will be something like this https://gist.github.com/kaareal/7a9eec01e6f86aa855678541201ae638
```

Comment:
FYI This error seems to actually be coming from @bedrockio/config.
The resolver tries to load the webpack.config.js which loads @bedrockio/config -> this fails in your case as it's looking for .env at the root in your case instead of `services/web`.

Maybe that package could not throw a hard error when the file isn't found but instead just return an empty object for the config?

Reply:
maybe